applications/rubintv/templates/deployment-frontend.yaml

```diff
@@ -0,0 +1,80 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ template "rubintv.fullname" . }}-frontend
+  labels:
+    {{- include "rubintv.labels" . | nindent 4 }}
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      {{- include "rubintv.selectorLabels" . | nindent 6 }}
+  strategy:
+    type: "Recreate"
+  template:
+    metadata:
+      {{- with .Values.frontend.podAnnotations }}
+      annotations:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      labels:
+        {{- include "rubintv.selectorLabels" . | nindent 8 }}
+    spec:
+      automountServiceAccountToken: false
+      containers:
+        - name: {{ .Chart.Name }}-frontend
+          env:
+            - name: "ENVIRONMENT_URL"
+              value: {{ .Values.global.baseUrl }}
+            - name: "SAFIR_PATH_PREFIX"
+              value: {{ .Values.frontend.pathPrefix | quote }}
+            {{- if (not .Values.frontend.debug) }}
+            - name: "SAFIR_PROFILE"
+              value: "production"
+            {{- end }}
+            {{- with .Values.frontend.env }}
+            {{- toYaml . | nindent 12 }}
+            {{- end }}
+          image: "{{ .Values.frontend.image.repository }}:{{ .Values.frontend.image.tag | default .Chart.AppVersion }}"
+          imagePullPolicy: {{ .Values.frontend.image.pullPolicy | quote }}
+          ports:
+            - name: "http"
+              containerPort: 8080
+              protocol: "TCP"
+          readinessProbe:
+            httpGet:
+              path: "/"
+              port: "http"
```

Comment:
We didn't do this previously because getting `/` is expensive. Is that still the case?

Reply:
I don't know, actually.  We're doing this on the JupyterLab controller as a readiness probe and I thought it was how we were planning on doing FastAPI healthchecks.

Reply:
I double-checked and it looks like this is hitting a metadata endpoint, so we should be okay.